functions/index.js

```diff
@@ -4,6 +4,37 @@ const cors = require('cors')({origin: true});
 const admin = require('firebase-admin');
 admin.initializeApp();
 
+exports.saveAudioSuggestions = functions.https.onRequest(async (req, res) => {
+  const uuidv1 = require('uuid/v1');
```

Comment:
Should this be in the function or outside of it?

Reply:
You mean the require? I didn't want to interrupt other people deploying cloud functions because of this package is missing, so I just put the require inside. But I can put it outside of course.